src/middlewares/logger.middleware.ts

```diff
@@ -0,0 +1,25 @@
+import { Injectable, Logger, NestMiddleware } from '@nestjs/common';
+import { Request, Response, NextFunction } from 'express';
+import { JwtService } from '@nestjs/jwt';
+
+@Injectable()
+export class LoggerMiddleware implements NestMiddleware {
+  private readonly logger = new Logger();
+  private readonly jwt = new JwtService();
```

Comment:
PullProofAI

Consider injecting the `JwtService` through the constructor instead of creating a new instance within the middleware.
```diff
- private readonly jwt = new JwtService();
+ constructor(private readonly jwt: JwtService) {}
```



<!-- This is an auto-generated comment by PullProofAI -->